pkg/chartmuseum/server/multitenant/handlers.go

```diff
@@ -107,8 +108,20 @@ func (server *MultiTenantServer) getStorageObjectRequestHandler(c *gin.Context)
 
 func (server *MultiTenantServer) getAllChartsRequestHandler(c *gin.Context) {
 	repo := c.Param("repo")
+	offsetString := c.DefaultQuery("offset","0")
+	offset, convErr := strconv.Atoi(offsetString)
+	if convErr != nil || offset < 0 {
+		c.JSON(400, gin.H{"error": "offset is not a valid non-negative integer"})
+		return
+	}
+	limitString := c.DefaultQuery("limit","20")
+	limit, convErr := strconv.Atoi(limitString)
+	if convErr != nil || limit <= 0 {
+		c.JSON(400, gin.H{"error": "limit is not a valid positive integer"})
+		return
+	}
 	log := server.Logger.ContextLoggingFn(c)
-	allCharts, err := server.getAllCharts(log, repo)
+	allCharts, err := server.getAllCharts(log, repo, offset, limit)
```

Comment:
As I understand this, the default values of `offset` and `limit` are respectively `0` and `20`. So if I don't change all my current queries (which do not have these query parameters), I will now get these limits, and this is a backward incompatible change.

Reply:
@davidovich Please check the changes now. I fixed it by setting default limit to "-1"